src/Parser/LASParser.js

```diff
@@ -137,16 +150,22 @@ export default {
             console.warn("Warning: options 'skip' not supported anymore");
         }
 
-        const input = options.in;
+        const crsIn = options.in?.crs || 'EPSG:3857';
+        const crsOut = options.out?.crs || crsIn;
 
         const lasLoader = await loader();
         const parsedData = await lasLoader.parseFile(Transfer(data), {
-            colorDepth: input?.colorDepth,
+            colorDepth: options.in?.colorDepth,
+            crsIn,
+            crsOut,
+            projDefs: {
+                [crsIn]: proj4.defs(crsIn),
+                [crsOut]: proj4.defs(crsOut),
+            },
```

Comment:
could you use the same structure in/out :
```js
const options = {
      in: {
         crs: '....' // instead of crsIn
         colorDepth: '...'
         projDefs: '....'
      },
      out: {
         crs: '....' // instead of crsOut
         colorDepth: '...'
         projDefs: '....'
      }
   }
}
```